packages/@eventual/core-runtime/src/local/local-container.ts

```diff
@@ -212,19 +226,34 @@ export class LocalContainer {
       () => this.localConnector.getTime()
     );
     this.timerClient = new LocalTimerClient(this.localConnector);
-    this.executionHistoryStore = new LocalExecutionHistoryStore();
+    this.executionHistoryStore = localPersistanceStore.register(
+      "execution-history",
+      (data) => LocalExecutionHistoryStore.fromSerializedData(data)
+    );
     this.taskProvider = props.taskProvider ?? new GlobalTaskProvider();
-    this.taskStore = new LocalTaskStore();
+    this.taskStore = localPersistanceStore.register("tasks", (data) =>
+      LocalTaskStore.fromSerializedData(data)
+    );
     this.subscriptionProvider =
       props.subscriptionProvider ?? new GlobalSubscriptionProvider();
     this.entityProvider = new GlobalEntityProvider();
-    const entityStore = new LocalEntityStore({
-      entityProvider: this.entityProvider,
-      localConnector: this.localConnector,
-    });
-    const bucketStore = new LocalBucketStore({
-      localConnector: this.localConnector,
-    });
+    const entityStore = localPersistanceStore.register("entities", (data) =>
+      LocalEntityStore.fromSerializedData(
+        {
+          entityProvider: this.entityProvider,
+          localConnector: this.localConnector,
+        },
+        data
+      )
+    );
+    const bucketStore = localPersistanceStore.register("buckets", (data) =>
```

Comment:
I'd imagine buckets should be built on top of the file system always since they could be large.

Reply:
probably, will add later